spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -19,14 +19,14 @@
     let(:user) { FactoryBot.create(:user) }
     let(:headers) { 'ID,Title,Title (Original Script),Author,Author (Original Script),Format,Language,Published/Created,Date,Description,Series,Location,Call Number,Notes' }
 
-    let(:data1) { ['9741216', 'Adriaen van de Velde : Dutch master of landscape', 'Cornelis, Bart', 'Book', 'English', 'London: Paul Holberton Publishing', '2016', '224 pages : illustrations (some color) ; 29 cm', 'Marquand Library', 'ND653.V414 A4 2016', 'Published to accompany the exhibition held at the Rijksmuseum, Amsterdam, 24 June - 15 September 2016 and Dulwich Picture Gallery, London, 12 October 2016 - 15 January 2017.'] }
-    let(:data2) { ['3148879', 'Sin, Chae-hong', '신재홍', "Tongnipkun u\u0306i cho\u0306nt'u", '독립군의전투　/'] }
+    let(:data1) { ['9997412163506421', 'Adriaen van de Velde : Dutch master of landscape', 'Cornelis, Bart', 'Book', 'English', 'London: Paul Holberton Publishing', '2016', '224 pages : illustrations (some color) ; 29 cm', 'Remote Storage: Marquand Use Only', 'ND653.V414 A4 2016', 'Published to accompany the exhibition held at the Rijksmuseum, Amsterdam, 24 June - 15 September 2016 and Dulwich Picture Gallery, London, 12 October 2016 - 15 January 2017.'] }
+    let(:data2) { ['9935444363506421', 'Replacement migration : is it a solution to declining and ageing populations?', 'Book', 'English', 'New York: United Nations', '2001', 'viii, 151 p. : ill. ; 28 cm.', 'United Nations Collection; Wallace Hall', '01.XIII.19; JV6225 .R464 2001'] }
     let(:bad_value) { 'Adriaen van de Velde : Dutch master of landscape / Bart Cornelis, Marijn Schapelhouman' }
 
     it 'renders a CSV list of metadata' do
       sign_in user
-      user.bookmarks.create!([{ document_id: '9741216', document_type: 'SolrDocument' }])
-      user.bookmarks.create!([{ document_id: '3148879', document_type: 'SolrDocument' }])
+      user.bookmarks.create!([{ document_id: '9997412163506421', document_type: 'SolrDocument' }])
+      user.bookmarks.create!([{ document_id: '9935444363506421', document_type: 'SolrDocument' }])
```

Comment:
Question: Why is this not 9931488793506421?

Reply:
This is the same test. The csv is generated and the info is there. I preferred to test English and not Korean.